projects/wp-nextjs/tsconfig.json

```diff
@@ -13,8 +12,8 @@
     "noEmit": true,
     "incremental": true,
     "esModuleInterop": true,
-    "module": "esnext",
-    "moduleResolution": "node16",
```

Comment:
No need for uppercasing `esnext` here.
Question on `moduleResolution: node` - is there a particular reason you've gone backwards to it from `node16`? ATM the recommended options are `nodenext` or `bundler`.

Reply:
mainly bc node is the default one that next.js sets. With `node16` it was also requiring me to include the js extension when doing dynamic imports which felt confusing. I'll try nodenext and bundler.

Reply:
`bundler` did the trick, that's exactly what I needed.